tcms/xmlrpc/api/telemetry.py

```diff
@@ -0,0 +1,64 @@
+from modernrpc.core import rpc_method
```

Comment:
Move this file under the telemetry app.

Reply:
Done. What would be the appropriate name then? `tcms.telemetry.api.telemetry` looks weird.

Reply:
Just `telemetry/api.py`